src/site/content/en/blog/prefers-reduced-motion/index.md

```diff
@@ -203,10 +203,8 @@ reveal animations are gone, and just the regular scrolling motion is left. The s
 shows the demo in action:
 
 <figure>
-  <video muted playsinline intrinsicsize="1098x870" controls
-    poster="prefers-reduced-motion.png"
-  >
-    <source src="prefers-reduced-motion.mp4" type="video/mp4">
+  <video muted playsinline intrinsicsize="1098x870" controls poster="prefers-reduced-motion.png">
```

Comment:
`intrinsicsize` is deprecated (https://github.com/GoogleChrome/web.dev/pull/1937).